src/aca_main.cpp

```diff
@@ -139,8 +141,14 @@ int main(int argc, char *argv[])
   signal(SIGINT, aca_signal_handler);
   signal(SIGTERM, aca_signal_handler);
 
-  while ((option = getopt(argc, argv, "b:h:g:s:p:c:t:o:md")) != -1) {
+  while ((option = getopt(argc, argv, "a:b:h:g:s:p:c:t:o:md")) != -1) {
```

Comment:
let's make it pretty by ordering the list from:
"a:b:h:g:s:p:c:t:o:md"
to:
"a:p:b:h:g:s:c:t:o:md"

Reply:
fixed